detekt.yml

```diff
@@ -0,0 +1,3 @@
+style:
+  ReturnCount:
+    max: 3
```

Comment:
What does that mean?

Reply:
This means we can write functions like this:
```
fun method() {
  if (condition1) return
  if (condition2) return
  if (condition3) return
}
```

The default number is 2 and I think it's pretty restrictive. Tbh I'm fine with any number of early returns, especially in void methods; on a private project I'd just disable this rule altogether I guess.